Dockerfile

```diff
@@ -22,6 +22,18 @@ RUN apt-get update && apt-get install -y \
     iputils-ping \
     tmux
 
+ARG BEAKER_VERSION
```

Comment:
Can you also add this to `Dockerfile.olmo`? I'd like to keep the two dockerfiles synced up as much as possible for now.